manifests/r/RicoSuter/NSwagStudio/NSwag-Build-1155/RicoSuter.NSwagStudio.locale.en-US.yaml

```diff
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.0.2 $debug=QUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: RicoSuter.NSwagStudio
+PackageVersion: NSwag-Build-1155
```

Comment:
```suggestion
PackageVersion: 13.14.5.0
```

Reply:
cc @RicoSuter: Would it be possible for you to tag your GitHub releases with the version number from Control Panel in the future? That would be greatly appreciated :)

At the moment, Control Panel shows `13.14.5.0` instead of `NSwag-Build-1155`.